podman-desktop

```diff
@@ -57,14 +57,14 @@ test -f "$IDENTITY" || die "cannot find identity $IDENTITY"
 test -f "$PUBKEYFILE" || die "cannot find pubkey $PUBKEYFILE"
 PUBKEY=$(cat "$PUBKEYFILE")
 
-multipass set client.primary-name="$INSTANCE_NAME"
-
-echo >&2 "this may take a while - to view progress:
+PRIMARY=$(multipass get client.primary-name)
+if [ "$PRIMARY" = "primary" ]; then
```

Comment:
is this the default? I am wondering why we would override in this case but not in the general case

if it is the default, what is the best way for me to test that?

Reply:
k, found it in the docs https://multipass.run/docs/set-command